regression/tests/studio/studio_base_test.py

```diff
@@ -80,6 +80,9 @@ def clean_up_function(self):
             pages_page.wait_for_the_visibility_of_new_page()
             pages_page.delete_all_pages()
 
+        if pages_page.is_page_shown() is False:
```

Comment:
I don't think we should put this here. This method basically says "if it's not showing, just click the toggle button". That's smart programming, but for a test, you should know the expected state of the page. If it's not showing--but it should be...then you should encounter an error. 

Putting this kind of logic here could hide display bugs.